src/replace/module.coffee

```diff
@@ -10,10 +16,24 @@ module.exports = (path, stub) ->
   realThing = require(quibble.absolutify(path))
   fakeThing = imitate(realThing, path)
   quibble(path, wrapIfNeeded(fakeThing, realThing))
-  return fakeThing
+  return _.merge(fakeThing, imitateProperties(realThing))
 
 wrapIfNeeded = (fakeThing, realThing) ->
   if isConstructor(realThing)
     wrapWithConstructor(fakeThing)
   else
     fakeThing
+
+imitateProperties = (realThing) ->
+  return {} unless _.isFunction(realThing)
+  _.reduce imitatableKeys(realThing), wrapKey(realThing), {}
+
+imitatableKeys = (realThing) ->
+  _.keys(realThing).filter (key) ->
```

Comment:
Drop in [`_.keysIn`](https://lodash.com/docs/4.17.0#keysIn) instead if inherited keys are desired, as well.

I'll leave the "desirability" determination up to others, since I don't have any context around this.